apps/editor.planx.uk/src/@planx/components/Checklist/Editor/Editor.tsx

```diff
@@ -34,7 +34,11 @@ export const ChecklistEditor: React.FC<Props> = (props) => {
   const type = TYPES.Checklist;
 
   const formik = useFormik<ChecklistWithOptions>({
-    initialValues: parseChecklist({ ...props.node?.data, options: props?.options }),
+    initialValues: parseChecklist({
+      ...props.node?.data,
+      options: props?.options,
+      groupedOptions: props.groupedOptions,
```

Comment:
nit: I think this needs to be `props?.groupedOptions`? Or is it actually always guaranteed to be defined now (eg `[]` if "flat" options without groups rather than `undefined`)?

Reply:
Good spot - fixed on the rebase to make it clearer.